staging/vhost-device-console/src/vhu_console.rs

```diff
@@ -0,0 +1,1285 @@
+    ) -> JoinHandle<Result<()>> {
+        let vhu_console = Arc::clone(vhu_console);
+
+        let exit_eventfd = vhu_console.read().unwrap().exit_event.as_raw_fd();
+        // Spawn a new thread to handle input.
+        spawn(move || {
+            let term = Term::stdout();
+            let mut fdset = FdSet::new();
+            fdset.insert(term.as_raw_fd());
+            fdset.insert(exit_eventfd);
+            let max_fd = fdset.highest().expect("Failed to read fdset!") + 1;
+
+            loop {
+                let ready = vhu_console.read().unwrap().ready_to_write;
+                let exit = vhu_console.read().unwrap().controller.read().unwrap().exit;
+
+                if exit {
+                    dbg!("Exit!");
+                    break;
+                } else if ready {
+                    let mut fdset_clone = fdset;
+                    enable_raw_mode().expect("Raw mode error");
+
+                    match select(Some(max_fd), Some(&mut fdset_clone), None, None, None) {
+                        Ok(_num_fds) => {
+                            disable_raw_mode().expect("Raw mode error");
+                            let exit = vhu_console.read().unwrap().controller.read().unwrap().exit;
+                            if (fdset_clone.contains(exit_eventfd)) && exit {
+                                dbg!("Exit!");
+                                break;
+                            }
+
+                            if fdset_clone.contains(term.as_raw_fd()) {
+                                if let Some(character) = match term.read_key().unwrap() {
+                                    Key::Char(character) => Some(character),
+                                    Key::Enter => Some('\n'),
+                                    Key::Tab => Some('\t'),
+                                    Key::Backspace => Some('\u{8}'),
+                                    _ => None,
+                                } {
+                                    // Pass the data to vhu_console and trigger an EventFd
+                                    let input_buffer = character.to_string();
+                                    vhu_console
+                                        .write()
+                                        .unwrap()
+                                        .rx_data_fifo
+                                        .add(input_buffer)
+                                        .unwrap();
+                                    vhu_console.write().unwrap().rx_event.write(1).unwrap();
+                                }
+                            }
+                        }
+                        Err(e) => {
+                            disable_raw_mode().expect("Raw mode error");
+                            panic!("Error in select: {}", e);
+                        }
+                    }
+                }
+            }
+
+            Ok(())
+        })
+    }
+    pub fn kill_console_thread(&self) {
+        dbg!("Kill thread\n");
```

Comment:
I have this output:
```
[vhost-device-console/src/vhu_console.rs:626:9] "Kill thread\n" = "Kill thread\n"
```

```suggestion
        dbg!("Kill thread");
```

Can you also check other `dbg!()`, `trace!()`, etc. calls ?

Reply:
Of course, the latest implementation has deleted all the unnecessarry of the `trace` and `dbg` calls.

Reply:
Great, but for me it was also fine to leave them if they could be useful. As you prefer ;-)